ios/ReactNativeSampleApp.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,7 +2,7 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:ReactNativeSampleApp.xcodeproj">
+      location = "group:BitriseReactNativeSample.xcodeproj">
```

Comment:
Unrelated to this change: I don't know this for sure, but this seems like a file that should be `.gitignore`d. WDYT?

Reply:
Fair question. This file defines how Xcode opens and handles the project, so we do want it to be checked in.